### PR TITLE
New setup for strong-pm [redhat]

### PR DESCRIPTION
**Pull request by xuqingfeng**
Don't merge, just for review.

### CODE REVIEW

File: tasks/install_redhat.yml [hunk 1]
```diff
@@ -14,19 +14,35 @@
...
-    enabled=true
+- name: Redhat | Create custom config folder
+  file:
+    name: /etc/systemd/system/strong-pm.service.d
+    state: directory
+    mode: 0755
 
-- name: Redhat | Start Strongloop Process Manager service
+- name: Redhat | Add strong-pm.service custom config
+  templaete:
```
Comment (zbal) [inline tasks/install_redhat.yml]: typo

File: templates/etc_systemd_system_strong-pm.service.d_custom.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/opt/node-{{ node_version }}-linux-x64/bin/node /usr/lib/node_modules/strong-pm/bin/sl-pm.js --listen 8701 --base /var/lib/strong-pm --base-port 3000 --driver direct {% if strong_pm_skip_default_install %} --skip-default-install {% endif %}}
```
Comment (zbal) [inline templates/etc_systemd_system_strong-pm.service.d_custom.conf.j2]: We can't hardcode the path like that - you want to have a step that register the `ExecStart` string and re-use it here; this is the only way to support various versions install path of `node`

**Review by zbal — COMMENTED** [review]
minor typos

Note - before we merge - we need to tag the current version of `master` as `v1.0.0` -- this one will become `v2.0.0rc1` (or whatever name)

**Comment by xuqingfeng** [comment]
Updated, don't know if it will work or not...

**Pull request closed without merging**